Mathlib/MeasureTheory/Function/LpSpace.lean

```diff
@@ -646,6 +646,10 @@ theorem Memℒp.memℒp_of_exponent_le_of_measure_support_ne_top
   have : Fact (μ (toMeasurable μ s) < ∞) := ⟨by simpa [lt_top_iff_ne_top] using hs⟩
   exact memℒp_of_exponent_le hfq hpq
 
+lemma Memℒp.mono_exponent [IsFiniteMeasure μ] {q : ℝ≥0∞} (hpq : p ≤ q) (hfq : Memℒp f q μ) :
```

Comment:
Is that the same statement as [MeasureTheory.Memℒp.memℒp_of_exponent_le](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/LpSeminorm/CompareExp.html#MeasureTheory.Mem%E2%84%92p.mem%E2%84%92p_of_exponent_le) ?

Reply:
Oh indeed! Are you happy for me to rename it to `Memℒp.mono_exponent`?

Reply:
yes, please do. And please golf the proof! (I think `memℒp_of_exponent_le_of_measure_support_ne_top` did not exist when that long proof was written).
Also `memℒp_of_exponent_le_of_measure_support_ne_top` should probably be moved to the CompareExp file since it is about comparing exponents.

Reply:
Sadly I can't golf the proof using `Memℒp.mono_exponent_of_measure_support_ne_top` since the proof of `Memℒp.mono_exponent_of_measure_support_ne_top` uses it